tests/logs.rs

```diff
@@ -180,16 +180,16 @@ fn test_debug_logs_raw() {
     assert!(error.times_called() > 0);
 
     // Confirm only the debug log file exists.
-    assert!(std::path::Path::new(DEBUG_LOG_FILE).exists());
-    assert!(!std::path::Path::new(METRICS_LOG_FILE).exists());
+    assert!(std::path::Path::new(DEBUG_FILE).exists());
+    assert!(!std::path::Path::new(METRICS_FILE).exists());
 
-    cleanup_files(METRICS_LOG_FILE, DEBUG_LOG_FILE);
+    cleanup_files(METRICS_FILE, DEBUG_FILE);
 }
 
 #[test]
 fn test_debug_logs_json() {
-    const METRICS_LOG_FILE: &str = "metrics-json2.log";
-    const DEBUG_LOG_FILE: &str = "debug-json2.log";
+    const METRICS_FILE: &str = "metrics-json2.log";
+    const DEBUG_FILE: &str = "debug-json2.log";
```

Comment:
I would personally do such large re-namings always in a different PR. (either before or after the other changes)

Reply:
I have split it out here: https://github.com/tag1consulting/goose/pull/151

Reply:
Now that #151 is merged, this has been rebased.